setup-build-freetype.py

```diff
@@ -0,0 +1,166 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+# This will build a FreeType binary for bundling with freetype-py on Windows,
+# Linux and macOS. The environment determines if it's a 32 or 64 bit build (on
+# Windows, set $env:PYTHON_ARCH="64" for a x64 build; on Linux, set it to "32"
+# to explicitly make a 32 bit build).
+
+# It can be used stand-alone, in conjunction with setup.py and on CI services
+# like Travis and Appveyor. You need CMake and some C/C++ compiler suite (e.g.
+# GCC, Visual Studio Community 2017, ...)
+
+import distutils.dir_util
+import glob
+import os
+import subprocess
+import sys
+from os import path
+
+FREETYPE_HOST = "https://download.savannah.gnu.org/releases/freetype/"
+FREETYPE_TARBALL = "freetype-2.9.tar.bz2"
+FREETYPE_URL = FREETYPE_HOST + FREETYPE_TARBALL
+FREETYPE_SHA256 = (
+    "e6ffba3c8cef93f557d1f767d7bc3dee860ac7a3aaff588a521e081bc36f4c8a")
+HARFBUZZ_HOST = "https://www.freedesktop.org/software/harfbuzz/release/"
+HARFBUZZ_TARBALL = "harfbuzz-1.7.6.tar.bz2"
+HARFBUZZ_URL = HARFBUZZ_HOST + HARFBUZZ_TARBALL
+HARFBUZZ_SHA256 = (
+    "da7bed39134826cd51e57c29f1dfbe342ccedb4f4773b1c951ff05ff383e2e9b")
+
+root_dir = "."
+build_dir = path.join(root_dir, "build")
+# CMake requires an absolute path to a prefix.
+prefix_dir = path.abspath(path.join(build_dir, "local"))
+build_dir_ft = path.join(build_dir, FREETYPE_TARBALL.split(".tar")[0], "build")
+build_dir_hb = path.join(build_dir, HARFBUZZ_TARBALL.split(".tar")[0], "build")
+
+CMAKE_GLOBAL_SWITCHES = ("-DCMAKE_COLOR_MAKEFILE:BOOL=false "
+                         "-DCMAKE_PREFIX_PATH:PATH=\"{}\" "
+                         "-DCMAKE_INSTALL_PREFIX:PATH=\"{}\" ").format(
+                             prefix_dir, prefix_dir)
+
+bitness = None
```

Comment:
Where is this “bitness” variable used?

Reply:
Further down in step 3 when deciding whether to turn on `-fPIC` or not.

Reply:
Oh I see, thanks